data/abilities.ts

```diff
@@ -258,7 +258,7 @@ export const Abilities: {[abilityid: string]: AbilityData} = {
 			if (effect?.effectType !== 'Move') {
 				return;
 			}
-			if (source.species.id === 'greninja' && source.hp && !source.transformed && source.side.foe.pokemonLeft) {
+			if (source.species.id === 'greninja' && source.hp && !source.transformed && source.side.getFoePokemonLeft()) {
```

Comment:
```suggestion
			if (source.species.id === 'greninja' && source.hp && !source.transformed && source.getFoePokemonLeft()) {
```

There's no point in making this a function if it's in also based off of `Side` and is longer than just typing `side.foe.pokemonLeft`